pkg/webui/console/containers/side-bar/index.js

```diff
@@ -55,39 +68,41 @@ const Sidebar = ({ isDrawerOpen }) => {
 
   const sidebarClassnames = classnames(
     style.sidebar,
-    'd-flex pos-fixed direction-column j-between gap-cs-m bg-tts-primary-050',
+    'd-flex direction-column j-between gap-cs-m bg-tts-primary-050',
     {
       [style.sidebarMinimized]: isMinimized,
-      [style.sidebarOpen]: isMobile && isDrawerOpen,
-      'p-cs-s': !isMinimized,
-      'p-vert-cs-m': isMinimized,
+      [style.sidebarOpen]: isDrawerOpen,
+      'p-cs-m': !isMinimized,
+      'p-vert-cs-s': isMinimized,
       'p-sides-cs-xs': isMinimized,
     },
   )
 
```

Comment:
This is still not optimal and quite confusing. All of this should rather be just done using plain CSS and media queries.